src/CommunityToolkit.Maui/Views/Expander/Expander.macios.cs

```diff
@@ -0,0 +1,55 @@
+using System.Reflection;
+using CoreGraphics;
+using UIKit;
+
+namespace CommunityToolkit.Maui.Views;
+
+public partial class Expander
+{
+	static void ForceUpdateCellSize(CollectionView collectionView, Size size, Point? tapLocation)
+	{
+		if (tapLocation is null)
+		{
+			return;
+		}
+
+		var controller = GetController(collectionView);
+
+		if (controller?.CollectionView.CollectionViewLayout is UICollectionViewFlowLayout layout)
+		{
+			var cells = layout.CollectionView.VisibleCells.OrderBy(x => x.Frame.Y).ToArray();
+			var clickedCell = GetCellByPoint(cells, new CGPoint(tapLocation.Value.X, tapLocation.Value.Y));
+			if (clickedCell is null)
+			{
+				return;
+			}
+
+			for (int i = 0; i < cells.Length; i++)
+			{
+				var cell = cells[i];
+
+				if (i > 0)
```

Comment:
is there a change to `i` be less than zero?

Reply:
i can be equal to 0